readthedocs/builds/querysets.py

```diff
@@ -103,7 +103,9 @@ def public(
             if user.is_superuser:
                 queryset = self.all()
             else:
-                queryset = self._add_from_user_projects(queryset, user)
+                queryset = self._add_from_user_projects(
+                    queryset, user, admin=True, member=True
```

Comment:
This matches what we have in .com, and actually in .com we override it completely (so this change doesn't affect .com),

https://github.com/readthedocs/readthedocs-corporate/blob/ad902765b50f570b44961ce8c8248e9fca713fb8/readthedocsinc/builds/querysets.py#L79-L84.

This was changed so the tests that I added with `RTD_ALLOW_ORGANIZATIONS=True` would pass on .org, otherwise this was returning an empty queryset (admin and member would be both false)

https://github.com/readthedocs/readthedocs.org/blob/adec99886dafd5fccba498bcab707b3a5cb0b40a/readthedocs/core/permissions.py#L47-L72

I thought I mentioned this in the PR description, but looks like I forgot, sorry